blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/EnumRollup.java

```diff
@@ -0,0 +1,41 @@
+package com.rackspacecloud.blueflood.types;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EnumRollup implements Rollup {
+    private Map<Long, Long> en2Value = new HashMap<Long, Long>();
+
+    public EnumRollup withObject(Long valueNameHashcode, Long value) {
```

Comment:
Don't we need to store the full string name here, and not just the hash code?  Other wise how does the query endpoint return it?